src/frontend/api/index.js

```diff
@@ -0,0 +1,24 @@
+const fetcher = (path, method, body) =>
+  new Promise(resolve => {
+    fetch(path, {
+      method,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(body),
+    }).then(res => {
+      if (!res.ok) {
+        throw new Error('요청 실패');
+      }
+      resolve(res.json());
+    });
+  });
+
+const API = {
+  get: path => fetcher(path, 'GET'),
+  post: (path, body) => fetcher(path, 'POST', body),
+  delete: path => fetcher(path, 'DELETE'),
+  patch: (path, body) => fetcher(path, 'PATCH', body),
```

Comment:
```suggestion
  put: (path, body) => fetcher(path, 'PUT', body),
```
저희가 `put`을 사용하기로 했으니 `put`으로 변경하면 좋을 것 같아요 ! ㅎㅎ